tox.ini

```
@@ -13,8 +13,12 @@ deps =
 
 [testenv:codestyle]
 deps = pycodestyle
-commands = pycodestyle --ignore=E722,E501 jdcal.py test_jdcal.py
+commands = pycodestyle --ignore=E722,E501 src/jdcal/jdcal.py tests/test_jdcal.py
 
 [testenv:docstyle]
 deps = pydocstyle
-commands = pydocstyle jdcal.py test_jdcal.py
+commands = pydocstyle src/jdcal/jdcal.py 
+
+[testenv:mypy]
+deps = mypy
+commands = mypy --strict src/jdcal/jdcal.py
```

We should type check the entire package, not just one file.

```suggestion
commands = mypy --strict src/
```